jsonmerge_utils.py

```diff
@@ -5,10 +5,23 @@
 @author: GRENTOR
 """
 import os
+import logging
 #from tqdm import tqdm
 from merge_files import Merge
-
-
+logger = logging.getLogger(__name__)
+def configure_logger():
+    """
+    configures the logger object
+    """
+    logger.setLevel(level=logging.INFO)
+    if not logger.handlers:
+        # Prevent logging from propagating to the root logger
+        logger.propagate = 0
+        log_formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+        stream_handler = logging.StreamHandler()
+        stream_handler.setFormatter(log_formatter)
+        logger.addHandler(stream_handler)
+configure_logger()

```

Comment:
this function should be called from the main() function within the try block
like this:
```
try:
    format_logger()
    data_dir = input('Folder Path: ')
```

Reply:
so you would like me to run the program and push a log with the PR?

Reply:
Surely you would have run it locally for testing the logging output, 
just attach/comment it in conversation here.

Reply:
Yes, I have just ran it locally for testing. 
After running the logger it looks like this:
```INFO:jsonmerge-logger:/Users/paulsin/Desktop/jsonmergeutils/SampleJsonfiles/tes1.json
INFO:jsonmerge-logger:/Users/paulsin/Desktop/jsonmergeutils/SampleJsonfiles/tes2.json
INFO:jsonmerge-logger:/Users/paulsin/Desktop/jsonmergeutils/SampleJsonfiles/tes3.json
INFO:jsonmerge-logger:/Users/paulsin/Desktop/jsonmergeutils/SampleJsonfiles/tes4.json
INFO:jsonmerge-logger:/Users/paulsin/Desktop/jsonmergeutils/SampleJsonfiles/tes5.json
INFO:jsonmerge-logger:/Users/paulsin/Desktop/jsonmergeutils/SampleJsonfiles/tes6.json
INFO:jsonmerge-logger:/Users/paulsin/Desktop/jsonmergeutils/SampleJsonfiles/tes7.json
INFO:jsonmerge-logger:/Users/paulsin/Desktop/jsonmergeutils/SampleJsonfiles/tes8.json
INFO:jsonmerge-logger:/Users/paulsin/Desktop/jsonmergeutils/SampleJsonfiles/tes9.json
INFO:jsonmerge-logger:/Users/paulsin/Desktop/jsonmergeutils/SampleJsonfiles/tes10.json
INFO:jsonmerge-logger:/Users/paulsin/Desktop/jsonmergeutils/SampleJsonfiles/tes11.json
INFO:jsonmerge-logger:/Users/paulsin/Desktop/jsonmergeutils/SampleJsonfiles/tes12.json
INFO:jsonmerge-logger:/Users/paulsin/Desktop/jsonmergeutils/SampleJsonfiles/tes13.json
INFO:jsonmerge-logger:/Users/paulsin/Desktop/jsonmergeutils/SampleJsonfiles/tes14.json
INFO:jsonmerge-logger:/Users/paulsin/Desktop/jsonmergeutils/SampleJsonfiles/tes15.json
INFO:jsonmerge-logger:/Users/paulsin/Desktop/jsonmergeutils/SampleJsonfiles/tes16.json
INFO:jsonmerge-logger:/Users/paulsin/Desktop/jsonmergeutils/SampleJsonfiles/tes17.json
INFO:jsonmerge-logger:/Users/paulsin/Desktop/jsonmergeutils/SampleJsonfiles/tes18.json
INFO:jsonmerge-logger:/Users/paulsin/Desktop/jsonmergeutils/SampleJsonfiles/tes19.json
INFO:jsonmerge-logger:/Users/paulsin/Desktop/jsonmergeutils/SampleJsonfiles/tes20.json
INFO:jsonmerge-logger:/Users/paulsin/Desktop/jsonmergeutils/SampleJsonfiles/tes21.json
INFO:jsonmerge-logger:/Users/paulsin/Desktop/jsonmergeutils/SampleJsonfiles/tes22.json
INFO:jsonmerge-logger:/Users/paulsin/Desktop/jsonmergeutils/SampleJsonfiles/tes23.json
INFO:jsonmerge-logger:/Users/paulsin/Desktop/jsonmergeutils/SampleJsonfiles/tes24.json
INFO:jsonmerge-logger:Done!!
INFO:jsonmerge-logger:24 Files Processed
INFO:jsonmerge-logger:12 Result files created
```

Reply:
the format for logger:
`log_formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')`

doesnt seem to be reflected in the output

But this much is ok for  now I guess